R/module_tabs_with_filters.R

```diff
@@ -29,15 +30,16 @@
 NULL
 
 #' @rdname module_tabs_with_filters
-ui_tabs_with_filters <- function(id, modules, datasets, filter = teal_slices()) {
+ui_tabs_with_filters <- function(id, modules, datasets, filter = teal_slices(), progress) {
```

Comment:
what if `ui_tabs_with_filters` gets a parameter called `progress_modules` and `modules_datasets` gets parameter called `progress_data`, so that we can differentiate between various progress types used here? I see we have 2 progress objects in the type: one for data, one for modules.

Reply:
I don't follow. These two functions are different scopes, what is the benefit of using different variable names?

Reply:
My point was that it's hard to understand if those are the same progresses used across the package if you look for `progress` term with a global search/find

<img width="887" alt="image" src="https://github.com/insightsengineering/teal/assets/133694481/97eee590-13ae-4905-9dcc-c58bc2e919c8">

Reply:
It's not a big deal, I'm fine having the same name as it's shorter and within the scope of one function is not confusing.

Reply:
Would you do the same for `filter` ,`modules`, `datasets` or `datanames`?

Reply:
Forget about it. I would not do that for other modules like modules, filters or datasets